mrp_production_grouped_by_product/models/stock_picking_type.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class StockPickingType(models.Model):
+    _inherit = 'stock.picking.type'
+
+    mo_grouping_max_hour = fields.Integer(
```

Comment:
Why don't use Datetime field to allow set minutes and avoid set/show UTC time?

Reply:
Because it includes a date, which will confuse the user.